### ISSUE TITLE
[Bug] No personal and no patients after update to 0.65.1

### ISSUE THREAD

**Issue by abuech2s** [CorsixTH/CorsixTH#2039]
#### Describe the issue
After downloading and installing 0.65.1 I have started a new game based on the first level.
Even after "1 year" of gameplay, I did not get any patients and there is no personal to buy.

Dont know what to do ...

#### Steps to Reproduce
1. *Load save game attached to this ticket*

#### Save Game

Altern[0651_Bug.zip](https://github.com/CorsixTH/CorsixTH/files/7072894/0651_Bug.zip)

#### System Information
**CorsixTH Version:** 0.65.1

**Operating System:** *e.g. Windows 10*

**Theme Hospital Version:** *GOG.com,*

#### Gamelog.txt

[gamelog.txt](https://github.com/CorsixTH/CorsixTH/files/7072889/gamelog.txt)
[config.txt](https://github.com/CorsixTH/CorsixTH/files/7072890/config.txt)

### Additional Info
*Paste any screen shots or other additional information that might help
illustrate the problem.*

**Comment by lewri** [comment]
Just pulling out the error we need
```lua
Error in timer handler: 
Lua\entities\humanoid.lua:623: Can only finish current action
stack traceback:
	[C]: in function 'assert'
	Lua\entities\humanoid.lua:623: in method 'finishAction'
	Lua\humanoid_actions\answer_call.lua:40: in field '?'
	Lua\entities\humanoid.lua:536: in upvalue 'Humanoid_startAction'
	Lua\entities\humanoid.lua:628: in method 'finishAction'
	Lua\humanoid_actions\walk.lua:193: in local 'timer_function'
	Lua\entity.lua:216: in field 'tick'
	Lua\entities\humanoids\staff.lua:105: in method 'tick'
	Lua\world.lua:1042: in method 'onTick'
	Lua\app.lua:1111: in function <Lua\app.lua:1108>
	(...tail calls...)
	Lua\app.lua:1012: in function <Lua\app.lua:1007>
```

**Comment by lewri** [comment]
Just to also check:

Have you built a reception, hired receptionist, built a gp, and hired a doctor?

If the tutorial is still trying to progress have you ran though it?

**Comment by abuech2s** [comment]
Yes, I build a reception.
If "gp" means "general practical room" (or something like this), then yes.
Was not able to hire anyone, 4 times "0" for available persons.

Did not make the tutorial - I know the game.

**Comment by abuech2s** [comment]
uploaded the gameplay (*.sav file) above

**Comment by abuech2s** [comment]
![screenshot](https://user-images.githubusercontent.com/44176089/131262867-8e1e70e3-bd6e-496e-8f3c-ab67c9acd6b2.jpg)

**Comment by abuech2s** [comment]
Further information:
Started the game with: "Introduction: off" and "Difficulty: Middle"
It happens in case, that I open the hospital "manually" or even waiting until the timer the gone.

From the beginning of the game the available persons are always 0.

**Comment by lewri** [comment]
@mugmuggy any thoughts? Sadly not around to dig into the save

**Comment by lewri** [comment]
Curious. As soon as I load your save I get staff in the hire screen
![image](https://user-images.githubusercontent.com/20030128/131384376-dfe50d76-3ff8-4794-abc5-3836732ca188.png)

**Comment by abuech2s** [comment]
Yes I play with german language.
I will check your file and change the language.
Then I let you know ...

**Comment by mugmuggy** [comment]
My only thought was that no-one was hired. That gamelog appears to apply to another save where patients have arrived and have arrived, waited and left.

**Comment by abuech2s** [comment]
Haha. I am so sorry guys. I think I did not play for a long time... I think I fooled myself -.-
The whole time, I mixed up the button of buying new personal and the button next to it seeing the current personal state. I really thought I could buy personal at the window, at which I made my screenshot.

In other words: Everything works fine. Its your choice to let this ticket open (based on the stack above) or to close it :D
